frontend/src/modules/Maintenance/components/MaintenanceViewer.js

```diff
@@ -314,11 +309,32 @@ export const MaintenanceViewer = () => {
     useState(START_MAINTENANCE);
   const [maintenanceWindowStatus, setMaintenanceWindowStatus] =
     useState(INACTIVE_STATUS);
+  const [maintenanceModes, setMaintenanceModes] = useState([]);
   const [dropDownStatus, setDropDownStatus] = useState(false);
   const [refreshingTimer, setRefreshingTimer] = useState('');
   const { enqueueSnackbar, closeSnackbar } = useSnackbar();
   const dispatch = useDispatch();
 
+  const fetchMaintenanceModes = useCallback(async () => {
```

Comment:
where do you use the maintenanceModes state?

Reply:
We should wait for client to be initialised before using this query.
I'm not really front end dev, so may be it's not necessary.